platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/BaseMarkerViewOptions.java

```diff
@@ -14,7 +14,10 @@
  *
  * @param <U> Type of the marker view to be composed.
  * @param <T> Type of the builder to be used for composing.
+ * @deprecated Use a {@link com.mapbox.mapboxsdk.style.layers.SymbolLayer} instead. An example of converting Android
+ * SDK views to be used as a symbol see https://gist.github.com/tobrun/349777a6ef0dfb55245a544344ee197a.
```

Comment:
How about we instead include this approach as an activity in the TestApp while we work on making it available as a plugin?

Reply:
I added SymbolGeneratorActivity to the testapp, replaced all references to the gist with a permalink to an Activity in the testapp.

Reply:
@tobrun 👍

Reply:
If you are on-board with this change, could you re-review?